jni_gen_test/run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Prefer using a Dart script. E.g. a `setup.dart` with `Process.run`s.

Reply:
I am actually not using it anywhere but local machine :). On CI, each `dart run` is separate named step.

Reply:
Well, I was running it as well, because I did not want to have to figure out in which order to do things and what steps to run! 😆 

It would be nice if we could just `cd jni_gen`, `dart test` and then everything needed for the setup of the tests is automatically run.